media-libs/libsdl3/libsdl3-3.2.4.recipe

```diff
@@ -0,0 +1,77 @@
+SUMMARY="Simple Direct Media Layer 3.0"
+DESCRIPTION="Simple DirectMedia Layer is a cross-platform development library \
+designed to provide low level access to audio, keyboard, mouse, joystick, and \
+graphics hardware via OpenGL and Direct3D. It is used by video playback \
+software, emulators, and popular games."
+HOMEPAGE="https://www.libsdl.org/"
+COPYRIGHT="1997-2025 Sam Lantinga"
+LICENSE="Zlib"
+REVISION="1"
+SOURCE_URI="https://www.libsdl.org/release/SDL3-$portVersion.tar.gz"
+CHECKSUM_SHA256="2938328317301dfbe30176d79c251733aa5e7ec5c436c800b99ed4da7adcb0f0"
+SOURCE_DIR="SDL3-$portVersion"
+# PATCHES=""
+
+ARCHITECTURES="all !x86_gcc2"
+SECONDARY_ARCHITECTURES="x86"
+
+libVersion="$portVersion"
+libVersionCompat="$libVersion compat >= ${libVersion%%.*}"
+
+PROVIDES="
+    libsdl3$secondaryArchSuffix = $libVersionCompat
+    lib:libsdl3$secondaryArchSuffix = $libVersionCompat
+    "
+REQUIRES="
+    haiku$secondaryArchSuffix
+    lib:libGL$secondaryArchSuffix
+    "
+
+PROVIDES_devel="
+    libSDL3${secondaryArchSuffix}_devel = $portVersion
+    devel:libSDL3$secondaryArchSuffix = $libVersionCompat
+    devel:libSDL3_test$secondaryArchSuffix = $portVersion
```

Comment:
does this have the same SO version as libSDL3? Then you can reuse `$libVersionCompat` here, otherwise it should have its own version variable and use that.

Reply:
Uhm, it's a static library

Reply:
I think $portVersion is okay in this case. Sorry for the confusion.

Reply:
No problem :)